apidoc/Titanium/UI/Android/FloatingActionButton.yml

```
@@ -0,0 +1,67 @@
+---
+name: Titanium.UI.Android.FloatingActionButton
+summary: Androids Floating Action Button (FAB)
```

```suggestion
summary_ A floating action button (FAB) is a circular button that triggers the primary action in your app's UI.
```
(from the Android docs)